quaint/src/visitor/mysql.rs

```diff
@@ -562,6 +562,35 @@ impl<'a> Visitor<'a> for Mysql<'a> {
         Ok(())
     }
 
+    #[cfg(any(feature = "postgresql", feature = "mysql"))]
+    fn visit_json_array_agg(&mut self, array_agg: JsonArrayAgg<'a>) -> visitor::Result {
+        self.write("JSON_ARRAYAGG")?;
+        self.surround_with("(", ")", |s| s.visit_expression(*array_agg.expr))?;
+
+        Ok(())
+    }
+
+    #[cfg(any(feature = "postgresql", feature = "mysql"))]
+    fn visit_json_build_object(&mut self, build_obj: JsonBuildObject<'a>) -> visitor::Result {
+        let len = build_obj.exprs.len();
+
+        self.write("JSON_OBJECT")?;
+        self.surround_with("(", ")", |s| {
+            for (i, (name, expr)) in build_obj.exprs.into_iter().enumerate() {
+                s.visit_raw_value(Value::text(name))?;
+                s.write(", ")?;
+                s.visit_expression(expr)?;
+                if i < (len - 1) {
+                    s.write(", ")?;
+                }
+            }
+
+            Ok(())
+        })?;
+
+        Ok(())
+    }
```

Comment:
I think there's been some misunderstanding here. This PR shouldn't contain work about MySQL. It's not really harmful but it's irrelevant for the bug, right?

Reply:
The reason I wanted to check how it behaved with MySQL is to ensure that _when_ we actually tackle https://github.com/prisma/team-orm/issues/455, that we don't get into the same dead-end

Reply:
Yeah this is definitely a bit premature and out of scope here, I guess we were discussing different providers and my mind just went there without thinking 😅
Not really harmful indeed but I'd probably move it out to a separate branch so we don't have unused and untested code lying around and only keep this PR only about fixing the bug.